### DISCUSSION TITLE
Discount of 10% when book on day of event

### DISCUSSION THREAD

**Discussion by Konafets** [pretix/pretix#3419] [Q&A]
How can I offer a discount of 10% for a customer which buys a ticket on the same day as the event happens?

**Comment by raphaelm** [comment]
There's no  real automated way, you could use a different product "same day rate" that is only available on that day

**Comment by Konafets** [comment]
I figured it out by myself. I have two products: One with the normal price and one with a price discount. I activated both products for each date of an event and configured both products so that the normal product is available till one day before the event starts and the discounted product only on the day of the event.
